NYPL-discovery/discovery-front-end
# Add fix to fix use of discovery.nypl.org in redirect_uri

**What's this do?**
Fix all use of [qa-]discovery.nypl.org with [qa-]www.nypl.org in the redirect_uri for login.

**Why are we doing this? (w/ JIRA link if applicable)**
We've found that the login flow started by the DFE hold-request route builds a login redirect_uri that uses the internal discovery.nypl.org hostname. It should not.

**Do these changes have automated tests?**
None.

**How should this be QAed?**
After deployment to QA:
 - ensure logged out of QA RC
 - navigate to a hold request page [e.g. this](https://qa-www.nypl.org/research/research-catalog/hold/request/b15051798-i13728143/edd)
 - note that the redirect returned ('Location' header) features a redirect_uri with qa-www.nypl.org instead of qa-discovery.nypl.org. After logging in, you should arrive at qa-www.nypl.org instead of qa-discovery.nypl.org

**Dependencies for merging? Releasing to production?**
n/a

**Has the application documentation been updated for these changes?**

**Did someone actually run this code to verify it works?**
Depends on host header, so easiest to test in QA.
